index.html

```diff
@@ -83,6 +83,204 @@ <h1>
         </ul>
       </div>
       </section>
+
+      <!-- Tonic -->
+
+      <section class="grid">
+        <div class="flex">
+          <div class="work-card">
+            <img src="img/IMG1.png" alt="Tonic" />
+            <div class="work-card1">
+              <h2>Tonic</h2>
+              <div class="canopy">
+                <span>CANOPY</span>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p class="daily">
+                A daily selection of privately personalized reads; no accounts
```

Comment:
- [ ] In your project cards, kindly position your elements correctly. Adjust the padding and spacing to match the original design

![image](https://user-images.githubusercontent.com/98453979/219640368-0b9b1b50-e71e-48fc-b4cd-594488e347f3.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219640640-313f41fc-0a44-4a86-b6b7-8ad7d473ed24.png)